src/Hit/Utils.hs

```diff
@@ -0,0 +1,16 @@
+module Hit.Utils
```

Comment:
Let's rename this module to have a more clear name. We might want to move things here in the future. But I'm personally not a fan of too generic names.
```suggestion
module Hit.Formatting
```